### ISSUE TITLE
Hyprland crashes on start: m_pAqBackend fails or is null

### ISSUE THREAD

**Issue by pianocomposer321** [hyprwm/Hyprland#7299]
### Regression?

No

### System Info and Version

I can't run `hyprctl` since I'm not in hyprland. I'm running Linux Mint 22.04 with KDE Neon installed on top, and I'm installing hyprland using the nix package manager. My system is an HP Probook laptop (with integrated graphics).

Since I can't run `hyprctl` I don't know if that's all the information you need, so feel free to ask if you need more details.

### Description

As the title says, hyprland crashes on startup. If I install hyprland the regular way from nix (just using `nix profile install nixpkgs#hyprland`), this is the crash report:

[hyprlandCrashReport257367.txt](https://github.com/user-attachments/files/16588383/hyprlandCrashReport257367.txt)

After doing some research I found that for some people, building with the legacyrenderer option solved this, so I tried again, following [these steps](https://wiki.hyprland.org/Nix/Options-Overrides/#using-nix-repl). Hyprland still crashes, with a similar but different error message:

[hyprlandCrashReport201888.txt](https://github.com/user-attachments/files/16588415/hyprlandCrashReport201888.txt)

### How to reproduce

Just run hyprland using `nixGL Hyprland` (or `nixGL <path to hyprland executable>` if installed using the overrides).

**Comment by vaxerski** [comment]
you're missing gpu drivers (or they are misconfigured), this is not a hyprland issue

**Comment by pianocomposer321** [comment]
> you're missing gpu drivers (or they are misconfigured), this is not a hyprland issue

I'm not sure this is accurate... I thought this at first too, or maybe just that my graphics driver didn't support a recent enough version of opengl but when I ran `glxinfo` it showed that I had 4.x something. I saw somewhere that hyprland only requires opengl 3.2. Is this correct?

**Comment by vaxerski** [comment]
wait, are you running on a KDE session? Hyprland is not an x11 compositor and will refuse to run on x11. Read the last log:
```
[CRITICAL] m_pAqBackend couldn't start! This usually means aquamarine could not find a GPU or encountered some issues. Make sure you're running either on a tty or on a Wayland session, NOT an X11 one.
```

**Comment by pianocomposer321** [comment]
I'm running it from a tty, but I did try running it from a KDE Wayland session as well and that didn't work either. I run KDE on Wayland only, not on X11.

**Comment by vaxerski** [comment]
the log is not from a tty

**Comment by pianocomposer321** [comment]
I apologize, it is possible that log was from the time I ran it from KDE, but if so, like I said it was on Wayland.

I'm currently at work and I don't have my laptop with me, but I can find one from running it from the tty when I get home if that would be helpful.

**Comment by vaxerski** [comment]
would be yea please do

**Comment by pianocomposer321** [comment]
Update on my end, I decided to finally pull the trigger on trying out NixOS, and I was able to get hyprland working there. I'm not surprised that the issue was related to my using Mint, but I do apologize to you and to anyone who may stumble on this later that I wasn't able to keep debugging it.

I keep my home partition so I believe I still have the log files around if you want me to send more to you, but since I can't test it anymore I can't guarantee that I can find one specifically from running it in the TTY. If you'd like me to try that I can, otherwise if you'd like to just close the issue that's fine with me too
